### PR TITLE
added support for Alfred 3.1.1

### PR DESCRIPTION
**Pull request by sineld**

### CODE REVIEW

**Comment by innocarpe** [comment]
@stuartcryan This must be merged!

File: custom_iterm_script_iterm_3.1.1.applescript [hunk 1]
```diff
@@ -0,0 +1,45 @@
+-- This is v0.6 of the custom script for AlfredApp for iTerm 2.9+
```
Comment (rendhalver) [inline custom_iterm_script_iterm_3.1.1.applescript]: You should probably change the header to update the version and include a URL so when it's copied into iTerm people will know where it came from.

Comment (sineld) [reply custom_iterm_script_iterm_3.1.1.applescript]: I'm not sure how to do it.
Will you please update it as you have declared?

Comment (rendhalver) [reply custom_iterm_script_iterm_3.1.1.applescript]: Change this text to include the URL to this file.

**Comment by sineld** [comment]
@rendhalver done.

**Pull request merged**